src/pages/paperwork/[rescueId]/index.js

```diff
@@ -301,15 +301,14 @@ class Paperwork extends React.Component {
           }
 
           {
-            rescue.attributes.data?.markedForDeletion?.marked && (
+            rescue.attributes.outcome === 'purge' && (
               <div className="md-group">
                 <div className="marked-for-deletion">{'Marked for Deletion'}</div>
                 <div className="md-reason">
-                  {`"${rescue.attributes.data.markedForDeletion?.reason}"`}
-                  <div className="md-reporter">{` - ${rescue.attributes.data.markedForDeletion.reporter}`}</div>
```

Comment:
Is there no way for us to identify the last edited person / md reporter?
Though I question the usefulness of using the last edited person because that is not necessarily the md reporter (e.g. reopened cases), having something there is likely better than nothing.

Reply:
no nothing is listed in the rescue itself. it's saved behind the scenes in information we don't have access to